modules/siddhi-core/src/test/java/io/siddhi/core/query/FilterTestCase1.java

```diff
@@ -2895,4 +2896,41 @@ public void receive(long timeStamp, Event[] inEvents, Event[] removeEvents) {
         siddhiAppRuntime.shutdown();
     }
 
```

Comment:
@mayuravaani , @pcnfernando  asked you to remove only the unnecessary line spaces. That does not meant that you shouldn't have any line space. We should have line spaces for each logical code blocks. Check below comments.